scripts/train.py

```diff
@@ -316,24 +394,41 @@ async def run_training(
     job_type = "train_model"
 
     # Validate parameter dependencies
-    validate_params(track_and_upload, aws_env)
+    validate_params(track_and_upload=track_and_upload, aws_env=aws_env)
+
+    concept = await scripts.get_concept.get_concept_async(
+        wikibase_id=wikibase_id,
+        include_labels_from_subconcepts=True,
+        include_recursive_has_subconcept=True,
+        wikibase_config=wikibase_config,
+    )
+
+    if classifier_type:
+        classifier = create_classifier(
+            concept=concept,
+            classifier_type=classifier_type,
+            classifier_kwargs=classifier_kwargs or {},
+        )
+    else:
+        classifier = ClassifierFactory.create(concept)
```

Comment:
idea: would it be neater if we moved the class override _into_ the `ClassifierFactory` itself? ie rather than having this if/else in the script, we could just say:

```py
classifier = ClassifierFactory.create(
    concept,
    override_class=classifier_type  # either an actual value specified by the user, or None
    override_kwargs=classifier_kwargs  # as above
)
```

i feel like the kwarg and class name matching functionality are particularly well suited to living in the `knowledge_graph/classifier/__init__.py` compared to the single purpose script